bpf/node_config.h

```diff
@@ -191,7 +191,7 @@ DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x
 #define CILIUM_LB_AFFINITY_MAP_MAX_ENTRIES	65536
 #define CILIUM_LB_REV_NAT_MAP_MAX_ENTRIES	65536
 #define CILIUM_LB_MAGLEV_MAP_MAX_ENTRIES	65536
-#define CILIUM_LB_SKIP_MAP_MAX_ENTRIES		65536
+#define CILIUM_LB_SKIP_MAP_MAX_ENTRIES		100
```

Comment:
One of the previous commits sets this to 64k then this is set to 100. Was this intentional?

Reply:
This is the right size as we expect smaller number of entries in the map. I'll make it consistent.